packages/celotool/src/cmds/generate/private-key-to-address.ts

```diff
@@ -0,0 +1,23 @@
+/* tslint:disable no-console */
```

Comment:
Just renamed this file which used to be `account-address.ts`

Reply:
With this rename, we might have to hunt down other places in the codebase that currently calls this command, it might be easier to rename your "new" command